app/app.py

```diff
@@ -75,6 +75,14 @@ def find_recipe_in_url_window_algo_based():
 
     return create_json_response(ingred_paragraph, instr_paragraph, title, url)
 
+@app.route('/populate_training_example_from_url/', methods=['POST'])
+def populate_training_example_from_url():
+    password = request.form.get('password')
+
+    if password != "toy_password":
+        return {'error': 'Invalid password'}, 401
+
+    return 'OK', 200
```

Comment:
this is here because you didn't start a new feature branch.
the common practice is that each ticket require a new feature branch.
many ways to solve it.
you can do the following:
1- rename the branch to "<ticket-id>-<your-name>_<name_of_feature>" [our new convention from now on]
2- rebase on to master (read how), if there is a conflict on app.py take the changes from master.
3- squash the. commits (or just make sure there is only 1 commits)